### PR TITLE
Command line call stuff + vagrant fixes

### PR DESCRIPTION
**Pull request by MJJoyce**

### CODE REVIEW

File: src/main/java/gov/nasa/jpl/memex/pooledtimeseries/PoT.java [hunk 1]
```diff
@@ -28,18 +28,17 @@
...
 public class PoT {
 	
 	public static int frame_width = 320;
 	public static int frame_height = 240;
 
 	public static void main(String[] args) {
-		// TODO Auto-generated method stub
-
-		System.loadLibrary( Core.NATIVE_LIBRARY_NAME );
+        //System.loadLibrary( Core.NATIVE_LIBRARY_NAME );
```
Comment (chrismattmann) [inline src/main/java/gov/nasa/jpl/memex/pooledtimeseries/PoT.java]: so this line works for me? I don't think we should comment it out. Can you squash the rest of commits into a new PR and pull this one out?

**Comment by MJJoyce** [comment]
+1 Chris, requests where changed. Push includes updates to the README with additional information from Ryoo. I also added a comment to the loadLibrary call to note that at the moment the other hard coded is needed for the Vagrant build.

**Comment by chrismattmann** [comment]
@MJJoyce see what I did in #3 - I think this cleans it up.

**Comment by MJJoyce** [comment]
+1 good stuff. I'll close this

**Pull request closed without merging**